docs/guides/getting_started.md

```diff
@@ -386,8 +386,8 @@ const { URL } = require('url');
 
 // ...
 
-const ourDomain = 'https://apify.com';
-const absoluteUrls = links.map(link => new URL(link, ourDomain));
```

Comment:
The links would point to dead URLs if anybody forgets to update `ourDomain`

Reply:
Yeah, I like the change, but please explain `request.loadedUrl`. I think this is now the first time it's used in the tutorial

Reply:
Done.